Charades_Game_In_JavaScript/Charades Game In JavaScript final blog.md

```diff
@@ -0,0 +1,348 @@
+# Charades Game In JavaScript
+
+Quarantine life has been bored without hanging out with friends and buying boba. Everyone is desperate to have some fun at home. I recently read a post about creating a [memery game using Vanilla JavaScript by Marina Ferreira](https://medium.com/free-code-camp/vanilla-javascript-tutorial-build-a-memory-game-in-30-minutes-e542c4447eae), and found the techniques she used are very interesting, so I decided to create another popular game - Charades, using similar techniques and adding more features. You would dive a little more into some JavaScript methods in this tutorial. Let's get start to build our own charades game while learning HTML/CSS and JavaScript together in this one hour!
+
+### Live Demo: [Charades!](https://)
+## Download Starter Code
+Before we start, you can find all the codes here:  [Charades_Game_in_JavaScript Repo](https://github.com/bitprj/Charades_Game_In_JavaScript). 
+You can download the folder **"starter"** directly, or you can also start from scratch by creating **5 HTML files, 3 CSS files, and 4 JavaScript files in one single folder.**
+
+*Don't get scared by the amount! 2 of the HTML files are just the starting and ending page which are very simple, and their corresponding CSS and JavaScript will have a lot in common!*
+
+All the **HTML** files have the same starting template:
+```
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+
+  <title>Charades</title>
+
+  <link rel="stylesheet" href="{{YOUR_CSS_NAME}}">
+</head>
+<body>
+
+  <script src="{{YOURJAVASCRIPTNAME}}"></script>
+</body>
+</html>
+```
+
+All **CSS** files also have the same template to start with, as we are simply setting the background color and layout here:
+```
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+  }
+  
+  body {
+    height: 100vh;
+    display: flex;
+    background: #f8e5a6;
+  }
+```
+
+**A friendly tip**: personally, I use [Visual Studio Code](https://code.visualstudio.com/) to write all my codes, since it has colors and makes it easier to read. You could try download it if you don't know where to start!
+
+Now that we have all our files created, we could move on to the first step -- creating the starting and ending page!
+
+![](https://i.imgur.com/9VQL74c.jpg)
+
+
+
+## Starting Page and Ending Page
+### Ending Page
+In the [live demo](https://), we could see that the ending page is just a page with one single line of words to show a finishing message.
```

Comment:
do you want put the starting page before ending page? if not that's fine, just a suggestion